client/my-sites/plans-features-main/index.tsx

```diff
@@ -280,6 +286,8 @@ const PlansFeaturesMain = ( {
 		paidDomainName
 	);
 
+	const isFreeDomainFreePlanModalActive = config.isEnabled( 'onboarding-pm/free-free-modal' );
```

Comment:
Non-blocking/Nit:
```suggestion
	const isFreeDomainFreePlanModalEnabled = config.isEnabled( 'onboarding-pm/free-free-modal' );
```

(since active means something that is already present 🤔 )